sphinx_automodapi/automodsumm.py

```diff
@@ -294,7 +294,8 @@ def automodsumm_to_autosummary_lines(fn, app):
         # Note: we use __name__ here instead of just writing the module name in
         #       case this extension is bundled into another package
         from . import automodapi
-        if automodapi.__name__ in app._extensions:
+
+        if automodapi.__name__ in getattr(app, '_extensions', 'extensions'):
```

Comment:
Is this possible?
```
try:
    extensions = app.extensions
except AttributeError:  # Sphinx<1.6
    extensions = app._extensions
```
First trying `app._extensions` means that by default we're still accessing Sphinx internals and this could break at any time, e.g. if the Sphinx guys decide to change the private `app._extensions` in the future.